_posts/2018-02-13-Galaxy-User-Conf.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Program for the Galaxy User Conference
+location: Freiburg, Germany
+site: freiburg
+supporters:
+ - denbi
+ - elixir
+ - ViCE
```

Comment:
one of these was an event previously I thought? I think they are both already in the events folder.
![auswahl_345](https://user-images.githubusercontent.com/458683/36790014-b2029bd4-1c93-11e8-9bf9-03d160552d7a.png)

Reply:
They were post in the old version and were lost for unknown reason.

Reply:
But now we have duplicates? The events and the posts are more or less equivalent, there is a nice table in the post (is it updated relative to the google doc?)

In the event we link to the google doc. (I tried to embed it but no one responded.)